osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -186,18 +187,20 @@ private void load(OsuGameBase game, OsuColour colours, AudioManager audio)
 
             if (ShowTotalMultiplier)
             {
-                MainAreaContent.Add(new Container
+                MainAreaContent.Add(new FillFlowContainer
                 {
                     Anchor = Anchor.TopRight,
                     Origin = Anchor.TopRight,
                     AutoSizeAxes = Axes.X,
+                    Direction = FillDirection.Horizontal,
                     Height = DifficultyMultiplierDisplay.HEIGHT,
                     Margin = new MarginPadding { Horizontal = 100 },
-                    Child = multiplierDisplay = new DifficultyMultiplierDisplay
+                    Spacing = new Vector2(5, 0),
+                    Children = new Drawable[]
                     {
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre
-                    },
+                        new BpmDisplay(),
```

Comment:
If the BPM display is to be steered by `ShowTotalMultiplier` then that property needs a rename. Maybe something like `ShowModEffects` or something, with associated proper xmldoc to explain what the "effects" are.